src/nets/e2e_asr_attctc_th.py

```diff
@@ -118,33 +120,47 @@ def forward(self, x):
         alpha = self.mtlalpha
         if alpha == 0:
             self.loss = loss_att
-            loss_att_data = loss_att.data[0]
+            loss_att_data = loss_att.data[0] if torch_is_old else float(loss_att)
```

Comment:
Do we need to use `loss_att.item()`? if this implementation works fine, it is no problem.

Reply:
I prefer this style because it indicates the type.